src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,55 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event.
+ */
+public class Event extends Task {
+    private final LocalDateTime at;
+
+    public Event(String description, LocalDateTime at) {
+        this(description, at, false);
+    }
+
+    /**
+     * The event constructor.
+     *
+     * @param description The description of the event.
+     * @param at The date/time the event is due by.
+     * @param isDone The boolean keeping track of whether the event is done.
+     */
+    public Event(String description, LocalDateTime at, boolean isDone) {
```

Comment:
You may want to consider to use "eventTime" instead of "at"? Personally I feel it's more meaningful